### PR TITLE
New release 1.5.0

### LINKED ISSUE #109: ibm.power_aix.nim should be able to define NIM resources
**Is your feature request related to a problem? Please describe.**
When setting up or maintaining a NIM server with the power_aix collection, there is no way to define resources from existing files or from other resources.

**Describe the solution you'd like**
I would like to be able to use power_aix.nim to define at least this subset of the NIM object types:
  - lpp_source
  - spot
  - bosinst_data
  - exclude_files
  - mksysb
  - script
  - fb_script
  - res_group (I never used to use these, but it's currently *required* to actually perform an install with power_aix.nim)

I am working around this as described below, but need to begin removing my work-around code as we get closer to production deployments.

**Describe alternatives you've considered**
I'm currently using the **command** module to run *nim -o define*, and burning YAML to make it as idempotent as possible. This method requires the end-user to perform duties in the play that are better implemented inside the module.

**Additional context**
I am happy to implement this, but want an issue tracking it to avoid duplicate effort.

A generic *define* operation will eventually cause a breaking change, as *define_script* should be subsumed by it. The initial implementation can stub *define_script* to the new, internal, generic *define* logic. I will need to be directed on how that deprecation should work.

My initial idea is to add the rest of the object type keys as parallels to lpp_source, script, etc., and use them appropriately for *define* (and eventually *bos_inst* and other operations). I **think** there is always a polarity so that this would be well-defined. For example, one can define a spot from an lpp_source, but not the other way around.

### PR DESCRIPTION
**Pull request by pvtorres**
* New role nim_alt_disk_migration to automate the AIX software migration (upgrades) using an alternate disk.
  *  Information: https://github.com/IBM/ansible-power-aix/blob/dev-collection/roles/nim_alt_disk_migration/README.md
  * New module nim_resource to create, remove or display NIM resource objects such as lpp_source, spot, etc.
  * New enhanced nim module with new option "show" to display NIM object information.
  * New module tunables to automate Kernel Tuning management of no, nfso, vmo, ioo, raso and schedo.
  * New module tunfile_mgnt to automate Kernel Tuning with files containing sets of tuning parameters values: no, nfs, vmo, ioo, raso and schedo.
  * New enhanced inventory information for lpar_facts such full as os level, inc_core_crypto, nxcrypto, processor type/mode are more.
  * New enhanced inventory collection for lpp_facts additional info such as fixes (apar, SP, TL), version concistency (lppchk).
  * Fix DNF bootstrap not to download the AIX Toolbox bundle if it exist in the controller.
  * Updates to sanity tests.

### CODE REVIEW

**Review by GreenRodger8 — APPROVED** [review]
Looks good

File: docs/source/release_notes.rst [hunk 1]
```diff
@@ -5,6 +5,34 @@
...
+Version 1.5.0
+-------------
+Notes
+  * New role nim_alt_disk_migration to automate the AIX software migration (upgrades) using an alternate disk.
+  *  Information: https://github.com/IBM/ansible-power-aix/blob/dev-collection/roles/nim_alt_disk_migration/README.md
+  * New module nim_resource to create, remove or display NIM resource objects such as lpp_source, spot, etc.
+  * New enhanced nim module with new option "show" to display NIM object information.
+  * New module tunables to automate Kernel Tuning management of no, nfso, vmo, ioo, raso and schedo.
+  * New module tunfile_mgnt to automate Kernel Tuning with files containing sets of tuning parameters values: no, nfs, vmo, ioo, raso and schedo.
+  * New enhanced inventory information for lpar_facts such full as os level, inc_core_crypto, nxcrypto, processor type/mode are more.
```
Comment (jdejoya17) [inline docs/source/release_notes.rst]: typo,
... such as full os level.
instead of
... such full as os level

File: docs/source/release_notes.rst [hunk 2]
```diff
@@ -5,6 +5,34 @@
...
 
+Version 1.5.0
+-------------
+Notes
+  * New role nim_alt_disk_migration to automate the AIX software migration (upgrades) using an alternate disk.
+  *  Information: https://github.com/IBM/ansible-power-aix/blob/dev-collection/roles/nim_alt_disk_migration/README.md
+  * New module nim_resource to create, remove or display NIM resource objects such as lpp_source, spot, etc.
+  * New enhanced nim module with new option "show" to display NIM object information.
+  * New module tunables to automate Kernel Tuning management of no, nfso, vmo, ioo, raso and schedo.
+  * New module tunfile_mgnt to automate Kernel Tuning with files containing sets of tuning parameters values: no, nfs, vmo, ioo, raso and schedo.
```
Comment (jdejoya17) [inline docs/source/release_notes.rst]: suggestion:
... Kernel Tuning using files ...

File: docs/source/release_notes.rst [hunk 3]
```diff
@@ -5,6 +5,34 @@
 Releases
 ========
 
+Version 1.5.0
+-------------
+Notes
+  * New role nim_alt_disk_migration to automate the AIX software migration (upgrades) using an alternate disk.
```
Comment (jdejoya17) [inline docs/source/release_notes.rst]: New role, nim_alt_disk_migration, for automating AIX migration (upgrades) using a NIM server to create an alternate disk on a NIM client.

File: galaxy.yml [hunk 4]
```diff
@@ -16,7 +16,11 @@ readme: README.md
 # @nicks:irc/im.site#channel'
 authors:
 - Paul B. Finley (@pbfinley1911)
-
+- Pedro Torres (@pvtorres)
+- Nitish Mishra (@nitishmis)
+- Joseph de Joya (@dejoya17)
```
Comment (jdejoya17) [inline galaxy.yml]: typo
jdejoya17

**Review by jdejoya17 — CHANGES_REQUESTED** [review]

**Review by jdejoya17 — APPROVED** [review]

**Pull request merged**